src/intl/en.json

```diff
@@ -1283,11 +1283,11 @@
           "relevant_link_placeholder": "https://"
         }
       },
-      "personnel": {
-        "title": "Personnel",
-        "team_label": "Your team",
-        "team_detail": "Please list who will be working on this project, along with an explicit overview of their relevant skill sets and experience. You may provide links to portfolios or profiles to help the Decentraland community get to know who the DAO will be funding and how their backgrounds will contribute to your project’s success.",
-        "team_add_member": "Add team member",
+      "team": {
+        "title": "Team",
+        "members_label": "Members",
+        "members_detail": "Please list who will be working on this project and an explicit overview of their relevant skillset and experience. You may provide links to portfolios or profiles to help the Decentraland community get to know who the DAO will be funding and how their backgrounds will contribute to your project’s success.",
```

Comment:
Please list who will be working on this project and `include` an explicit overview of their relevant skillset and experience?

Reply:
I'll merge this but let's ask Gino and update it if necessary.